wayland/skel/.config/bunsen/autostart-wayland

```diff
@@ -0,0 +1,135 @@
+## Bunsenlabs User config files
+## All default BunsenLabs user config files are located in /usr/share/bunsen/skel.
+## The script bl-user-setup copies them to the user $HOME directory on first login.
+## See more info with command 'bl-user-setup --help'
+
+## BunsenLabs autostart
+## ====================
+## When you login to your BunsenLabs session, this autostart script
+## will be executed to launch any applications
+## you want to run at startup.
+##
+## NB, like openbox's and labwc's autostart script, this uses the default shell,
+## so "bashisms" are not supported.
+##
+## (Environment variables are set in ~/.config/bunsen/environment.)
```

Comment:
Environment variables are set in ~/.config/bunsen/environment-wayland

Reply:
..and that one